frontend/src/screens/RouteScreen.jsx

```diff
@@ -70,14 +128,13 @@ function RouteScreen(props) {
         </Toolbar>
       </AppBar>
 
-      <Paper>
-        <Box p={2} className="page-title">
-          {selectedRoute ? ` ${selectedRoute.title}` : null}
-          {direction ? ` > ${direction.title}` : null}
-          &nbsp;
-          {startStopInfo ? `(from ${startStopInfo.title}` : null}
-          {endStopInfo ? ` to ${endStopInfo.title})` : null}
-        </Box>
+      <Paper className={breadCrumbsWrapper}>
+         <Breadcrumbs separator={ <NavigateNextIcon fontSize="medium"  className={breadCrumbStyling}/> }>
```

Comment:
I think one of the warnings @exxonvaldez mentioned could be fixed like this 😀
```suggestion
         <Breadcrumbs separator={ <NavigateNextIcon fontSize="default"  className={breadCrumbStyling}/> }>
```

Reply:
👍 :)